tests/src/python/test_qgsserver_wms_getmap.py

```diff
@@ -1464,7 +1464,7 @@ def test_wms_getmap_filter_ogc_with_empty(self):
         self._img_diff_error(r, h, "WMS_GetMap_Filter_OGC")
 
         # empty filter
-        filter = '(<ogc:Filter xmlns="http://www.opengis.net/ogc">' "</ogc:Filter>)"
+        filter = '(<ogc:Filter xmlns:ogc="http://www.opengis.net/ogc">' "</ogc:Filter>)"
```

Comment:
Is this needed? I'm concerned that changing just to fix a test will result in real-world issues

Reply:
Yes, you're right. I added also a ogc namespace declaration

Reply:
Thanks @troopa81!